app/services/heading_service/heading_serialization_service.rb

```diff
@@ -51,12 +48,9 @@ def serialization_service
       if heading.declarable?
```

Comment:
Yeh, good point - I didn't depend upon it originally because it was all feature flagged but I can now

In general the loading logic is a bit odd because of how the caching is handled, we load outside the cache, then reload with a bunch of eager loading inside the cache block. I need to have a rethink about this, possibly using `yield` but in the mean time, yes using ns_declarable would definitely be beneficial